### PR TITLE
feat(motion): add marks.nvim improve nvim mark function

### PR DESCRIPTION
**Pull request by chaozwn**
add marks.nvim improve nvim mark function

### CODE REVIEW

File: lua/astrocommunity/motion/marks-nvim/init.lua [hunk 1]
```diff
@@ -0,0 +1,15 @@
+return {
+  "chentoast/marks.nvim",
+  event = "BufEnter",
```
Comment (Uzaaft) [inline lua/astrocommunity/motion/marks-nvim/init.lua:3]: Considering the different excluded filetypes, wouldn't `event = "User AstroFile"` make more sense?

Comment (chaozwn) [reply lua/astrocommunity/motion/marks-nvim/init.lua:3]: ok

**Review by mehalter — APPROVED** [review]
LGTM

**Comment by mehalter** [comment]
Not sure where the `editing-support` came from, but I went ahead and fixed the PR title to show the correct category

**Pull request merged**